containers/ddev-php-base/generic-files/usr/local/bin/install_php_extensions.sh

```diff
@@ -24,4 +24,5 @@ pkgs=$(yq ".${PHP_VERSION//.}.${ARCH} | join(\" \")" /etc/php-packages.yaml | aw
 # Echo the packages to be installed for logging
 echo "Installing packages for PHP ${PHP_VERSION} on ${ARCH}: $pkgs"
 
-apt-get -qq install --no-install-recommends --no-install-suggests -y $pkgs || exit $?
+# TODO: I'm not convinced that the apt-get update should be here or outside the script
+DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -qq install --no-install-recommends --no-install-suggests -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y $pkgs || exit $?
```

Comment:
Agreed, thanks.

Reply:
I tested it again without internet (to see the error) and `ddev start` tried to build it for 5 minutes until I stopped it.

Then I looked inside and saw that `ddev/ddev-webserver:20241105_less_php_versions` was not pushed after this change.

Reply:
Ah, there you go. I was focused on the other changes (in ddev code) and didn't even think about the fact that this was bundled in the container. Of course it could also be bundled in ddev, but then it would have to be copied in to use it.